storage/streams/streams_setup.sh

```diff
@@ -15,6 +15,7 @@ setupStreams() {
     ./kafka-server-start.sh $KAFKA_HOME/config/server.properties &
     sleep 2
     ./kafka-topics.sh --zookeeper localhost --create --topic storage-topic --partitions 1 --replication-factor 1  --config cleanup.policy=compact
+    ./kafka-topics.sh --zookeeper localhost --create --topic global-id-topic --partitions 1 --replication-factor 1 --config cleanup.policy=compact
```

Comment:
Ah yes - this is what I meant from above.  When we create the topics in code using the kafka AdminClient, we should make sure these default options are included.

Reply:
Maybe that's already happening, I didn't double check.  But IIRC it's not.